crypto/keys/multisig/multisig_test.go

```diff
@@ -259,10 +235,10 @@ func TestMultiSigMigration(t *testing.T) {
 
 	cdc := codec.NewLegacyAmino()
 
-	err := multisig.AddSignatureFromPubKey(multisignature, sigs[0], pkSet[0], pkSet)
+	require.NoError(t, multisig.AddSignatureFromPubKey(multisignature, sigs[0], pkSet[0], pkSet))
 
 	// create a StdSignature for msg, and convert it to sigV2
-	sig := legacytx.StdSignature{PubKey: pkSet[1], Signature: msg}
+	sig := legacytx.StdSignature{PubKey: pkSet[1], Signature: sigs[1].(*signing.SingleSignatureData).Signature}
```

Comment:
it was not failing because this line https://github.com/cosmos/cosmos-sdk/blob/b8d65dce17337edd8970293f5b01b89614e49e70/crypto/keys/multisig/multisig.go#L75 was returning nil instead of an error (and that was coming from the old multisig pub key code...).

I've already added a new test for this: https://github.com/cosmos/cosmos-sdk/blob/b8d65dce17337edd8970293f5b01b89614e49e70/crypto/keys/multisig/multisig_test.go#L180-L191